README.md

```diff
@@ -1,21 +1,82 @@
 # scalar-admin
 
-scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications.
+scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications. 
+It will help you to `PAUSE` and `UNPAUSE` the ledger to create point-in-time recovery (PITR). 
 
-## Use the client-side tool
+## scalar-admin client tool
+
+The scalar-admin client tool is used to manage the admin interface implemented applications.
```

Comment:
```suggestion
```

Reply:
Thank you for your suggestion.
Document updated with the above suggestion.